datahub-web-react/src/app/entity/shared/tabs/Entity/InputFieldsTab.tsx

```diff
@@ -26,17 +26,19 @@ export const InputFieldsTab = () => {
         return groupByFieldPath(ungroupedRows, { showKeySchema: false });
     }, [ungroupedRows]);
 
+    const editableSchemaMetadata: any = entityData?.editableInputFieldMetadata || undefined;
```

Comment:
big ol nit: I don't believe you need the `|| undefined` cuz if `entityData` is `undefined` so will `editableSchemaMetadata`